driver/native/native.go

```diff
@@ -139,8 +141,49 @@ func (r *parseResponse) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (d *Driver) writeRequest(ctx context.Context, req *parseRequest) error {
+	deadline, _ := ctx.Deadline()
+
+	sp, _ := opentracing.StartSpanFromContext(ctx, "bblfsh.native.Parse.encodeReq")
+	defer sp.Finish()
+
+	if !deadline.IsZero() {
```

Comment:
I think this'd be a little nicer with a defer:

```
if dl, _ := ctx.Deadline(); !dl.IsZero() {
   d.stdin.SetWriteDeadline(dl)
   defer d.stdin.SetWriteDeadline(time.Time{})
}
```

then you needn't have three separate blocks with semantic connections.